RELEASE_NOTES.md

```diff
@@ -1,3 +1,136 @@
+
+#### Kitchen DigitalOcean
+
+The Kitchen DigitalOcean driver has been updated from 0.10.5 to 0.11.0. This release adds slugs for Ubuntu 20.04 / RHEL 8 / Fedora 31 support, increases the the default instance memory size to 1GB, and adds support for VPCs. Thanks [@zmaupin](https://github.com/zmaupin), [@tolland](https://github.com/tolland), and [@gregf](https://github.com/gregf) for these improvements.
+
+#### Kitchen EC2
+
+The Kitchen EC2 driver has been updated from 3.3 to 3.6. This release lets the driver cleanly exit if the test instance was destroyed outside of the Test Kitchen run, either by automation or in the console. Test Kitchen will also now select the subnet with the most available IPs to better distribute systems across multiple Availability Zones. Thanks [@bdwyertech](http://github.com/bdwyertech) and [@kamaradclimber](http://github.com/kamaradclimber) for these improvements.
+
+#### Kitchen InSpec
+
+The Kitchen InSpec verifier has been updated to allow setting Chef InSpec plugins for use during the verification. This new functionality can be enabled by adding `load_plugins: true` to your InSpec verifier config. Thanks [@tecracer-theinen](https://github.com/tecracer-theinen) for this improvement.
+
+#### Kitchen Dokken
+
+The Kitchen Dokken driver has been updated from 2.8.1 to 2.9.0. This release adds a new provisioning configuration, `clean_dokken_sandbox`, that does not require cleaning the Chef Infra and Test Kitchen files between converges. This configuration will speed up repeatedly converging systems. This defaults to `true` which maintains the existing behavior. Thanks [@chrisUsick](https://github.com/chrisUsick)
+
+### Knife Plugins
+
+#### Knife Tidy
+
+Knife Tidy has been updated from 2.0.9 to 2.0.12 which provides compatibility with Chef Infra Client 15 and improves error handling in JSON parsing.
+
+#### Knife Azure
+
+Knife Azure was updated from 2.0.6 to 3.0.0 which includes significant performance enhancements.
+
+#### Knife EC2
+
+Knife EC2 has been updated from 1.0.28 to 2.0. This update resolves several errors bootstrapping nodes and avoids attempting to bootstrap nodes using private DNS which may not be accessible from the node running the bootstrap command.
+
+#### Knife Spork
+
+Knife Spork has been updated from 1.7.2 to 1.7.3. This release adds a new `--fail-if-frozen` flag to `knife spork check` to only fail when local version matches a frozen version and allows the git plugin to push to the current branch. Thanks to [@shoekstra](https://github.com/shoekstra) and [@zmaupin](https://github.com/zmaupin) for these improvements.
+
+#### Knife Windows
+
+Knife Windows has been updated from 3.0.6 to 4.0.2. This update includes significant performance improvements, fixes to for errors when using the concurrency flag, and better indication tha the legacy bootstrap commands have been replaced.
```

Comment:
@tas50 I sense something is a bit muddled for "fixes to for errors when using the concurrency flag".  Is there something missing, or. does "for" need some sort of formatting applied?